### PR TITLE
keep-dev/test: SSL

### PR DESCRIPTION
**Pull request by sthompson22**
### Intro

This turned it a bit of an adventure but here we are.  We're working on hardware wallet support for the dapp, as part of that we need a proper SSL connection.

I screwed around with self-signed certs for a bit which works but decided to go ahead and just put  ourselves behind Cloudflare. 

This took upgrading our Cloudflare setup to support custom hostnames, which we've done.  This does mean that we can't use our Google Hosted Zone subdomains for frontends that need secure connections, but I'll be looking into how to handle this via Terraform (in a later step).

#### keep-dev
- Moved the dapp deployment and services back to the `default` namespace.  Not directly related to the issue at hand, but was something in the backlog that made sense to do here.
- Moved the dapp deployment behind an ingress using Cloudflare
- Separated service/ingress configs into separate files.  We've been organizing configuration file scope around configuring a single object type for a given thing.  This is still experimental, may change at some point.

#### keep-test
- Moved the dapp deployment and services back to the `default` namespace.  Not directly related to the issue at hand, but was something in the backlog that made sense to do here.
- Moved the dapp deployment behind an ingress using Cloudflare.

### Testing

Both dev/test deployments have been shipped with the new configs, and aliased in Cloudflare.  Accessed both hosts from Chrome via `https` and `http` (to test redirect) and things look good:

- dapp.dev.tbtc.network

Screen Shot 2020-04-05 at 4 06 13 PM

- dapp.test.tbtc.network

Screen Shot 2020-04-05 at 4 07 00 PM

### CODE REVIEW

File: infrastructure/kube/keep-dev/tbtc-dapp-service.yaml [hunk 1]
```diff
@@ -3,15 +3,17 @@ apiVersion: v1
 kind: Service
 metadata:
   name: tbtc-dapp
-  namespace: tbtc
+  namespace: default
+  annotations:
```
Comment (Shadowfiend) [inline infrastructure/kube/keep-dev/tbtc-dapp-service.yaml:7]: Are these empty annotations?

File: infrastructure/kube/keep-dev/tbtc-dapp-service.yaml [hunk 2]
```diff
@@ -3,15 +3,17 @@ apiVersion: v1
...
   labels:
     app: tbtc
     type: dapp
 spec:
+  type: NodePort
```
Comment (Shadowfiend) [inline infrastructure/kube/keep-dev/tbtc-dapp-service.yaml:12]: For my own edification, what's the type if you don't specify it, as before? Or is it this, but we're wanting to be explicit?

Comment (sthompson22) [reply infrastructure/kube/keep-dev/tbtc-dapp-service.yaml:12]: `ClusterIp`, I've gone back and forth on being explicit even if we're using the default type.

File: infrastructure/kube/keep-dev/tbtc-dapp-service.yaml (hunk 1 above)
Comment (sthompson22) [reply infrastructure/kube/keep-dev/tbtc-dapp-service.yaml:7]: Nice catch, got rid of them.

**Pull request merged**